submissions/temitopeJohnson/html-forms/contactform.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Contact Form</title>
+    </head>
+    <body>
+        <h1>Contact Form Page</h1>
+        <form action="#" method="post" enctype="multipart/form-data">
+            <label for="fullname">Full Name:</label>
+            <input type="text" name="fullname" id="fullname" placeholder="Enter full-name"><br><br>
+            <label for="email">Email:</label>
+            <input type="email" name="email" id="email" placeholder="Enter email address"><br><br>
+            <label for="subject">Subject:</label>
+            <input type="text" name="subject" id="subject" placeholder="Enter Subject"><br><br>
+            <label for="message">Message:</label><br>
+            <textarea name="message" id="message" ></textarea><br><br>
```

Comment:
You should add the number of rows and columns. Effect this while doing your next task. (Adding page navigations)